README.md

```diff
@@ -73,13 +71,7 @@ export interface CreateIframeOptions {
   sessionId?: string
 
   /** The id of your organisation as issued to you by GetSetUp. */
-  embeddingOrgId: string
-
-  /**
-   * A stable id for the device the user is using to access the parent page. Used to report analytics back to your organisation.
-   * @deprecated Use {@link analyticsInfo} instead. This will be removed in a future version.
-   */
-  deviceId?: string
```

Comment:
Removed these from the readme because they are deprecated (still in the TSDocs on the interface itself though).